srcpkgs/mle/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'mle'
+pkgname=mle
+version=1.4.2
+revision=1
+build_style=gnu-makefile
+make_install_args="prefix=/usr"
+makedepends="libtool termbox-devel uthash lua-devel pcre-devel"
```

Comment:
These should be split between `hostmakedepends` and `makedepends`. As a rule of thumb, packages that contain binaries executed during the build go into host- while packages that contain header files needed for the build (-devel) go into makedepends.

Reply:
Thank you, these changes have been made. I was having some trouble understanding `hostmakedepends` and `makedepends` but I just reread the manual and I think I understand it better.